org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -642,17 +642,7 @@ SsoLlmDeveloper:
       - 'arn:aws:iam::aws:policy/AmazonBedrockFullAccess'
       - 'arn:aws:iam::aws:policy/AWSCloudFormationFullAccess'
 #   https://stackoverflow.com/questions/58125181/cloud-formation-cant-upload-template-file
-    inlinePolicy: >-
-      {
-          "Version": "2012-10-17",
-          "Statement": [
-              {
-                  "Effect": "Allow",
-                  "Action": "s3:*",
-                  "Resource": "arn:aws:s3:::cf-template*"
-              }
-          ]
-      }
+      - !Sub 'arn:aws:iam::${AWS::AccountId}:policy/SynapseLlmDeveloperPolicy'
```

Comment:
What's the deal with the comment just above?  does it pertain to this managed policy?

Reply:
Moved comment to definition of the managed policy